src/store/useManualRunStore.ts

```diff
@@ -7,17 +7,17 @@ import { createWithEqualityFn as create } from "zustand/traditional";
 
 import { StoreName } from "@enums";
 import { SessionsService } from "@services";
-import { ManualRunStore } from "@src/interfaces/store";
+import { ManualProjectData, ManualRunStore } from "@src/interfaces/store";
 
-const defaultProjectState = {
+const createDefaultProjectState = (): ManualProjectData => ({
```

Comment:
Why did you rewrite on function, as for me object better

Reply:
Doesn't really matter, changed back